packages/@lightningjs/ui-components/src/components/ListItem/ListItem.mdx

```diff
@@ -48,16 +59,14 @@ class Basic extends lng.Component {
 }
 ```
 
-<Canvas>
-  <Story id="components-listitem--list-item" />
-</Canvas>
-
 ## API
 
 ### Properties
 
 `ListItem` has the same properties as the ones listed in [Button](?path=/docs/components-button--basic) in addition to a couple listed below:
 
+<ArgTypes of={ListItemStories} />
```

Comment:
Should this be showing anything in the docs?
<img width="1017" alt="Screen Shot 2023-10-31 at 11 33 49 AM" src="https://github.com/rdkcentral/Lightning-UI-Components/assets/2678766/47617c1a-6959-4c87-99ca-39efaa56f1be">